packages/fax/src/rest/v3/callbacks/callbacks-webhook.ts

```diff
@@ -0,0 +1,66 @@
+import { CallbackProcessor } from '@sinch/sdk-client';
+import { IncomingHttpHeaders } from 'http';
+import { Fax, FaxCompletedEvent, IncomingFaxEvent } from '../../../models';
+
+export type FaxWebhookEventParsed = IncomingFaxEvent | FaxCompletedEvent;
+
+export class FaxCallbackWebhooks implements CallbackProcessor<FaxWebhookEventParsed> {
+
+  /**
+   * Reviver for a Fax Event
+   * This method ensures the object can be treated as a Fax Event and should be called before any action is taken to manipulate the object.
+   * @param {any} eventBody - the event body or form containing the Fax event notification.
+   * @return {FaxWebhookEventParsed} - The parsed Fax event object
+   */
+  public parseEvent(eventBody: any): FaxWebhookEventParsed {
```

Comment:
Is `any` necessary here?
Maybe it's worth giving `eventBody` a type?

Reply:
This is precisely the purpose of this method: an unknown object is received as an event. This method checks its properties to determine it can be processed as a Fax Event.